includes/polling/applications/memcached.inc.php

```diff
@@ -1,6 +1,53 @@
 <?php
 
-$data = $agent_data['app']['memcached'][$app['app_instance']];
+if (!empty($agent_data['app']['memcached'])) {
+    $data = $agent_data['app']['memcached'][$app['app_instance']];
+} else {
+    $options = '-O qv';
+    $oid     = 'nsExtendOutputFull.9.109.101.109.99.97.99.104.101.100';
+    $res     = explode(';' , snmp_get($device, $oid, $options));
```

Comment:
Had I known, that this is the textual representation of a serialized PHP object, then yes that's would have been drastically easier. Thanks for pointing it out. I guess I'll never stop learning.